script.js

```diff
@@ -27,56 +27,126 @@ const header = {
     }
   ]
 };
-
-webix.ui({
-  view: "popup",
-  id: "profile-popup",
-  body: {
-    view: "list",
-    data: [
-      { id: "1", label: "Settings" },
-      { id: "2", label: "Log out" }
-    ],
-    template: "#label#",
-    autoheight: true
-  }
+webix.ready(function() {
+  webix.ui({
+    view: "popup",
+    id: "profile-popup",
+    body: {
+      view: "list",
+      data: [
+        { id: "1", label: "Settings" },
+        { id: "2", label: "Log out" }
+      ],
+      template: "#label#",
+      autoheight: true
+    }
+  });
 });
 
 const form = {
   view: "form",
+  id: "addFilmForm",
   minWidth: 150,
   maxWidth: 250,
   elements: [
     { template: "EDIT FILMS", type: "section" },
     {
       view: "text",
-      label: "Title"
+      label: "Title",
+      name: "title",
+      invalidMessage: "Must be filled"
     },
     {
       view: "text",
-      label: "Year"
+      label: "Year",
+      name: "year",
+      invalidMessage: "1970 - the current year",
+      validate: webix.rules.isNumber
     },
     {
       view: "text",
-      label: "Rating"
+      label: "Rating",
+      name: "rating",
+      invalidMessage: "Number cannot be zero",
+      validate: webix.rules.isNumber
     },
     {
       view: "text",
-      label: "Votes"
+      label: "Votes",
+      name: "votes",
+      invalidMessage: "must be > 100000",
+      validate: webix.rules.isNumber
```

Comment:
don't mix `validate` property and the related form rule. choose either way to define validation, not both

Reply:
Ready to review